build.gradle

```diff
@@ -24,20 +32,29 @@ plugins {
     // See https://github.com/stevesaliman/gradle-properties-plugin
     // specify the env on the command line with:
     // gradle -PenvironmentName=x ...
-    id 'net.saliman.properties' version '1.4.6'
+    id 'net.saliman.properties' version '1.5.1'
 
     // This gradle plugin extends the ml-gradle plugin with
     // commands that make the Data Hub Framework do its magic
-    id 'com.marklogic.ml-data-hub' version '5.1.0'
+    id 'com.marklogic.ml-data-hub' version '5.8.0'
 }
 
 repositories {
     jcenter()
-    maven {url 'http://developer.marklogic.com/maven2/'}
+	// To use gradle in disconnected mode, you just need to set the 'disconnected' property. E.g. gradle compileJava -Pdisconnected
```

Comment:
Remove `jcenter()`, it's obsolete and has no impact.

Reply:
Nuked.